src/ui/sass/modules/_AlternativeVerticals.scss

```diff
@@ -70,6 +70,11 @@ $alternative-verticals-emphasized-font-weight: var(--yxt-font-weight-medium) !de
     }
   }
 
+  &-suggestionLink:focus, &-universalLink:focus {
```

Comment:
For the suggestionLink, it looks like there is currently logic to only underline the "yxt-AlternativeVerticals-suggestionLink--copy" on hover, should the same thing happen on focus?

For the universalLink, since the focus is the same as the hover CSS, could it be added below (currently L113)

Reply:
yeah that's a great suggestion, I was just going as close to the item's css as possible but what you said makes a lot of sense. The color is already set to --yxt-brand-primary too so there's really no reason to reinforce that on focus, either (I can only imagine this annoying somebody)